src/apps/HeyGPT/HeyGPT/HeyGPT.csproj

```diff
@@ -12,6 +12,11 @@
 		<EnableMsixTooling>true</EnableMsixTooling>
 		<WindowsAppSDKSelfContained>true</WindowsAppSDKSelfContained>
 		<DefaultLanguage>en-us</DefaultLanguage>
+		<IsSingleInstanceApp>true</IsSingleInstanceApp>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(IsSingleInstanceApp)'=='true'">
+		<DefineConstants>DISABLE_XAML_GENERATED_MAIN</DefineConstants>
```

Comment:
WindowsAppSDK creates a default main... In order to support single instance functionality we need to override and create our own.   This #define is what allows us to do that.